app/Http/Controllers/CompetitionsController.php

```diff
@@ -163,6 +163,18 @@ public function removeUser(Competition $competition, User $user)
      */
     public function message(Competition $competition, Contest $contest)
```

Comment:
Not sure if it matters, but you might not need to pass in `$contest`... the `getCompetitionOverview()` will append the contest as well and can be grabbed as `$competition->contest`, so this could be updated to:

```php
$messages = Message::where('contest_id', '=', $competition->contest->id)->get();
```